packages/shared/src/features/shortcuts/components/modals/CustomLinksModal.tsx

```diff
@@ -0,0 +1,179 @@
+    };
+  }, []);
+
+  return (
+    <Modal kind={Modal.Kind.FlexibleCenter} size={Modal.Size.Medium} {...props}>
+      <Modal.Header showCloseButton={false}>
+        <Typography tag={TypographyTag.H3} type={TypographyType.Body} bold>
+          Shortcuts
+        </Typography>
+        <div className="ml-auto flex gap-2">
+          <Button
+            variant={ButtonVariant.Float}
+            size={ButtonSize.Small}
+            onClick={() => {
+              props?.onRequestClose?.(undefined);
+            }}
+          >
+            Cancel
+          </Button>
+          <Button
+            variant={ButtonVariant.Primary}
+            size={ButtonSize.Small}
+            form="shortcuts-modal"
+            type="submit"
+          >
+            Save
+          </Button>
+        </div>
+      </Modal.Header>
+      <Modal.Body>
+        <form
+          ref={formRef}
+          id="shortcuts-modal"
+          className="flex flex-col gap-6"
+          onSubmit={async (event) => {
+            const { errors } = await onSaveChanges(event);
+            if (errors) {
+              return;
+            }
+
+            props?.onRequestClose?.(undefined);
+          }}
+        >
+          <div className="flex gap-4">
+            <div className="flex flex-1 flex-col gap-1">
+              <Typography bold type={TypographyType.Body}>
+                Show custom shortcuts
+              </Typography>
+
+              <Typography
+                type={TypographyType.Callout}
+                color={TypographyColor.Tertiary}
+              >
+                Enable custom shortcuts to easily access your favorite pages and
+                tools directly from daily.dev. Personalize your workflow and
+                navigate faster than ever.
+              </Typography>
+            </div>
+
+            <Switch
+              inputId="showTopSites-switch"
+              name="showTopSites"
+              className="w-20 justify-end"
+              compact={false}
+              checked={showTopSites}
+              onToggle={toggleShowTopSites}
+            >
+              {showTopSites ? 'On' : 'Off'}
+            </Switch>
+          </div>
+
+          <HorizontalSeparator />
+
+          <nav className="grid grid-cols-2 gap-10">
+            <CardSelection
+              title="My shortcuts"
+              description="Curate your own shortcuts manually"
+              icon={<UserIcon size={IconSize.XLarge} secondary={isManual} />}
+              isActive={isManual}
+              onClick={() => {
+                setIsManual(true);
+              }}
+            />
+            <CardSelection
+              title="Most visited sites"
+              description="Shortcuts are imported from your browser"
+              icon={
+                <SitesIcon
+                  size={IconSize.XLarge}
+                  secondary={isManual === false}
+                />
+              }
+              isActive={isManual === false}
```

Comment:
This was likely just copy-pasted, although  I see that there is a default value of `true` for the `isManual` variable. So I think we can just do `!isManual` when checking for false.

Reply:
Fixed